src/Microsoft.AspNetCore.Blazor.Server/Circuits/RemoteRenderer.cs

```diff
@@ -12,10 +14,17 @@ namespace Microsoft.AspNetCore.Blazor.Browser.Rendering
 {
     internal class RemoteRenderer : Renderer
     {
+        // The purpose of the timeout is just to ensure server resources are released at some
+        // point if the client disconnects without sending back an ACK after a render
+        private const int TimeoutMilliseconds = 60 * 1000;
```

Comment:
Hmmm is this the right place to fix this? 

I'm going to read on assuming that we need this here, but I'm wondering if this problem with crop up other places.

Reply:
Totally fine to have some way of configuring this at some point. Right now this is just a backstop to ensure we're not actually just leaking.